sqlglot/dataframe/sql/const.py

```diff
@@ -0,0 +1,10 @@
+JOIN_HINTS = {
```

Comment:
can we call it constants.py

Reply:
was a separate file necessary? could have just put it an the top where this was referenced